### ISSUE TITLE
No Iban?

### ISSUE THREAD

**Issue by dadifairman** [richnologies/ngx-stripe#38]
Hello,
Will you provide the 'iban' element soon?
Regards

**Comment by pradt** [comment]
Hi I'm looking for an IBAN option as well. Did you get through this ?

**Comment by dadifairman** [comment]
Nope I don't get through and using the common library for IBAN et ngx-stripe for credit card is not easy.

**Comment by pradt** [comment]
Ok, did you just import the Stripe's library and try and use that ? You wouldn't have any sample code that I could reference do you ?

**Comment by dadifairman** [comment]
Yes I tried to integrate the stripe service but with Angular6 it was difficult so i give up. I also tried the node module for stripe but it doesn't provide the same services like the ngx module. I removed all the code because i just poc it fast then i revert all the code. sorry

**Comment by pradt** [comment]
Ok no problem, I've managed to get it to work - however, I'm getting an error from Stripe regarding providing a type. Not sure whether it is due to the way I have implemented it, I'm still working through it. If you (or anyone reading this) is interested let me know.

**Comment by aliasbody** [comment]
I've added support for SEPA IBAN in a fork I'm adapting:
https://github.com/Blog-Ecommerce/ngx-stripe (this URL can change in the future)

**Comment by BRO-mesdocteurs** [comment]
Hi, i'm having trouble trying using IBAN element with ngx-stripe.
Is there an equivalent to "@ViewChild(StripeCardComponent) card: StripeCardComponent" for IBAN ?
Can you, please, give a snipet on the ts file side ?
